flask_admin/model/base.py

```diff
@@ -459,6 +459,31 @@ class MyModelView(BaseModelView):
         actions endpoints are accessible.
     """
 
+    column_list_row_actions = None
```

Comment:
It might be better if the attribute name indicated that these actions will be added to the standard per-row actions.

Maybe something like `extra_row_actions`, or `column_list_extra_row_actions` (probably too long)?